src/packages/fuelrats_api/v3/converters.py

```diff
@@ -1,11 +1,12 @@
 from __future__ import annotations
 
 import json
-from datetime import datetime
+
+import pendulum
+from pendulum import DateTime
```

Comment:
[style] Across the entire file: We should be consistent, is it: `import pendulum` and `pendulum.DateTime`,
or is it `from pendulum import DateTime` and `DateTime`?

Reply:
im going to use `import pendulum`, since potentially `pendulum.tz` is also needed...